File: src/main/java/com/willmadison/legacycodekatas/fulfillment/warehouse/consolidation/ConsolidatableOrder.java
package com.willmadison.legacycodekatas.fulfillment.warehouse.consolidation;

import java.time.LocalDateTime;
import java.util.Collection;

public class ConsolidatableOrder {

    public Collection<ConsolidatableOrderItem> items;

    public static class ConsolidatableOrderItem {

        public String id;

        public LocalDateTime lastUpdate;

        public boolean placed;
    }
}

File: src/main/kotlin/com/willmadison/legacycodekatas/fulfillment/warehouse/consolidation/ConsolidatableOrder.kt
package com.willmadison.legacycodekatas.fulfillment.warehouse.consolidation

import java.time.LocalDateTime

class ConsolidatableOrder(val items: Collection<ConsolidatableOrderItem>)

class ConsolidatableOrderItem(val id: String, var lastUpdate: LocalDateTime, val placed: Boolean) {
}